### PR TITLE
docs(dynamodb): add terraform template for dynamodb table

### PR DESCRIPTION
**Pull request by junlarsen**
## ☕️ Reasoning

Re-apply of https://github.com/nextauthjs/next-auth/pull/7586 per suggested by @ndom91  since that branch fell super far behind

I recently happened to use the DynamoDB adapter for NextAuth, and figured it could be helpful to add a template to the documentation, in line with the CDK and CloudFormation templates.

## 🧢 Checklist

- [x] Documentation
- [ ] Tests - N/A
- [ ] Ready to be merged

## 🎫 Affected issues

None

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Review by ndom91 — APPROVED** [review]
LGTM :+1:

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Left a small comment!

File: packages/adapter-dynamodb/src/index.ts [hunk 1]
```diff
@@ -156,6 +156,49 @@ export interface DynamoDBAdapterOptions {
  *       Enabled: true
  * ```
  *
+ * If you're using Terraform to manage your infrastructure, you can use this resource definition:
+ *
+ * ```hcl title=dynamodb.tf
+ * resource "aws_dynamodb_table" "next_auth" {
```
[the remark this answers is not in the document]
Comment (junlarsen) [reply packages/adapter-dynamodb/src/index.ts:162]: Of course. Would you also like me to re-name the CDK construct and CloudFormation template at the same time?

**Review by ndom91 — APPROVED** [review]
Awesome, thanks!

**Comment by ndom91** [comment]
Sorry but we're shipping new docs shortly and this change is already in, I'm going to close this PR :pray:

**Comment by junlarsen** [comment]
That's cool! Thanks for including it in the new docs

**Pull request closed without merging**